Tropos/Controllers/TRAppDelegate.m

```diff
@@ -38,6 +44,35 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
+- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken;
+{
+    NSString *channel = [[NSTimeZone defaultTimeZone] name];
```

Comment:
Given that we're not holding onto this reference, I believe that `+defaultTimeZone` and `+localTimeZone` are basically equivalent in this usage. However, I'm wondering if perhaps `+localTimeZone` would be more appropriate, semantically, for Tropos?